client-tez/src/test/java/org/apache/tez/runtime/library/common/sort/buffer/WriteBufferManagerTest.java

```diff
@@ -86,6 +98,20 @@ public void testWriteException() throws IOException, InterruptedException {
     int bitmapSplitNum = 1;
     int shuffleId = getShuffleId(tezTaskAttemptID, 1, 2);
 
+    Configuration conf = new Configuration();
+    FileSystem localFs = FileSystem.getLocal(conf);
+    Path workingDir = new Path(System.getProperty("test.build.data",
+        System.getProperty("java.io.tmpdir", "/tmp")),
```

Comment:
Could you use `org.junit.jupiter.api.io.TempDir` instead of `/tmp`? TempDir can be cleaned automatically. Maybe we don't have the permission of `/tmp` in some machine envrionment.

Reply:
> Could you use `org.junit.jupiter.api.io.TempDir` instead of `/tmp`? TempDir can be cleaned automatically. Maybe we don't have the permission of `/tmp` in some machine envrionment.

there need hadoop.fs.Path but TempDir only support nio.file.Path and io.File

Reply:
You can use similar code as below.
```
 public void test(@TempDir File tmpDir) throws IOException {
    Path workingDir = new Path(System.getProperty("test.build.data",
        System.getProperty("java.io.tmpdir", tmpDir.toString()))
}
```

Reply:
> tmpDir.toString()

thxs, i'll modify it later